assets/scripts/game.js

```diff
@@ -69,6 +69,18 @@ Game.prototype.createGameOverScene = function() {
 
 Game.prototype.killPlayer = function() {
   playerIsAlive = false;
```

Comment:
Does this still work if we move `playerIsAlive = false;` to below line [83][1]

[1]: https://github.com/technetos/SpaceRace/blob/418c7659538e13091d5e4ce941735f4b08ef5cde/assets/scripts/game.js#L83

Reply:
Yeah I'll make that change, this makes sense because we want the sound playing to have priority